### ISSUE TITLE
$ENV:EDITOR only works for nvim

### ISSUE THREAD

**Issue by hrai** [kelleyma49/PSFzf#249]
When I set vim as $ENV:EDITOR no editor is opened.

I only get the output below:
![image](https://github.com/kelleyma49/PSFzf/assets/4055444/63c5bf6f-3da8-4edd-9c5b-0fe9619c3c19)

https://github.com/kelleyma49/PSFzf/pull/158

But when I set it to `nvim` are per this https://github.com/kelleyma49/PSFzf/issues/157, I have similar issue as described here https://github.com/kelleyma49/PSFzf/issues/242:
![image](https://github.com/kelleyma49/PSFzf/assets/4055444/5eea8319-4932-44c6-98d6-649fa05247eb)

The editor is open but I can't trace what's going on in the editor.

**Comment by hrai** [comment]
Actually I identified the culprit to be `-NoNewWindow` flag that is a Windows bug.

https://github.com/kelleyma49/PSFzf/blob/f3e0ec83f1a2cf11c9337a1377601af68d816029/PSFzf.Functions.ps1#L136

Here's the command I ran that would give this error behaviour on Windows Terminal.
`Start-Process nvim -ArgumentList '"C:\_dev\yarn.lock" +0' -NoNewWindow`

@kelleyma49 is it possible to add a flag to the `FuzzyEdit` command to have this `-NoNewWindow` as optional?
